src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/ArrayTrailingCommaCheck.java

```diff
@@ -132,6 +139,27 @@
  *   ,};        // no violation
  * </pre>
  *
+ * <p>To configure check to always validate trailing comma:</p>
+ * <pre>
+ * &lt;module name="ArrayTrailingComma"&gt;
+ *   &lt;property name="alwaysDemandTrailingComma" value="true"/&gt;
+ * &lt;/module&gt;
+ * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * int[] a = new int[] {1, 2}; // violation, array should contain trailing comma.
+ * String[] b = new String[]{"foo", "bar",}; // ok
+ * int[] c = {1
+ *  2 // violation, array should contain trailing comma.
+ * };
+ * int[] d = {
+ * 1}; // violation, array should contain trailing comma.
+ * int[] c = {
+ *     1,
+ *     2 // violation, array should contain trailing comma.
```

Comment:
what is difference between this example and one in line 152?
also, field name 'c' is in use already

Reply:
Oh yeah, removed one of of them 👍 
Thanks